[32mINFO    [0m root:__init__.py:27 功能的名称为：test_ryvue 功能的描述为：None
[31m[1mERROR   [0m root:test_ryvue.py:70 menuManagement_006断言失败，用例数据为：{'parentId': 0, 'menuName': 'MENUMANAGEMENT_001', 'icon': '', 'menuType': 'M', 'orderNum': 1, 'isFrame': '1', 'isCache': '0', 'visible': '0', 'status': '0', 'path': 'menuManagement_001.com'}期望数据为：{'msg': "新增菜单'MENUMANAGEMENT_001'失败，菜单名称已存在", 'code': 500}服务器返回的期望数据：{"msg":"操作成功","code":200}
[31m[1mERROR   [0m root:__init__.py:33 断言失败